components/common/labels/List.tsx

```diff
@@ -28,3 +43,253 @@ export function LabelChip(props: ComponentProps<'span'>) {
+    labelDefFromService?.severity ||
+    labelDefFromService?.defaultSetting
+
+  const temporaryWarning = label.exp && (
+    <div className="flex flex-row items-start leading-4">
+      <ClockIcon className="h-4 w-4 mr-1" />
+      <p className="italic">
+        This is a temporary label and will expire at {label.exp}
+      </p>
+    </div>
+  )
+
+  const currentServiceReminder = isFromCurrentService && (
+    <div className="flex flex-row items-start leading-4">
+      <HomeIcon className="h-4 w-4 mr-1" />
+      <p className="italic">This label is from your own labeling service</p>
+    </div>
+  )
+
+  return (
+    <>
+      <div className="px-4 py-3">
+        {labelerProfile && (
+          <>
+            <h3 className="font-semibold text-gray-700 dark:text-gray-100 pb-1">
+              <a
+                href={buildBlueSkyAppUrl({ did: labelerProfile.did })}
+                target="_blank"
+                className="underline flex flex-row items-center"
+              >
+                {labelerProfile.displayName}
+                <ArrowTopRightOnSquareIcon className="h-3 w-3 ml-1" />
+              </a>
+            </h3>
+            <p className="leading-4 pb-3">{labelerProfile.description}</p>
+          </>
+        )}
+        {currentServiceReminder}
+        {temporaryWarning}
+      </div>
+
+      <div className="bg-gray-50 dark:bg-slate-600 px-4 py-3">
+        {labelDefFromService ? (
+          <>
+            <h4 className="font-semibold text-gray-700 dark:text-gray-100">
+              {labelDefInLocale?.name || label.val}
+            </h4>
+            {labelDefInLocale?.description && (
+              <p className="leading-4">{labelDefInLocale.description}</p>
+            )}
+            {hasPreferences && (
+              <ul className="pt-2">
+                {labelDefFromService.blurs ? (
+                  <li className="flex flex-row items-center">
+                    <EyeSlashIcon className="h-4 w-4 mr-1" /> Blurs{' '}
+                    {labelDefFromService.blurs}
+                  </li>
+                ) : null}
+                {labelDefFromService.severity ? (
+                  <li className="flex flex-row items-center">
+                    <ExclamationCircleIcon className="h-4 w-4 mr-1" /> Severity{' '}
+                    {labelDefFromService.severity}
+                  </li>
+                ) : null}
+                {labelDefFromService.defaultSetting ? (
+                  <li className="flex flex-row items-center">
+                    <CogIcon className="h-4 w-4 mr-1" />
+                    {`Default setting ${labelDefFromService.defaultSetting}`}
+                  </li>
+                ) : null}
+              </ul>
+            )}
+          </>
+        ) : (
+          <h4 className="leading-4">
+            <b>{label.val}</b> label does not have a custom definition. Users
+            will be able to configure the behavior of the label in app.
+          </h4>
```

Comment:
I'm not 100% sure whether this is accurate—we might want to check in with Paul/Bryan about the right way of messaging around this case.

Reply:
it's not. some labels are configurable in app though so changed the text from `will be" to "may be" which should have us covered.